src/diff.ts

```diff
@@ -0,0 +1,76 @@
+import { TextDocument, Range } from "vscode";
```

Comment:
Please add the copyright/license message

Reply:
Actually we are inconsistent with those licensing messages and we can probably phase them out since there is a LICENSE.txt in the root folder. Other extensions don't have per-file copyright. @hdpoliveira @mrcrowl What do you think?

Reply:
oh I thought this was the common practice. If there are other files without the message then it's ok for this one not to have it.
Can you please file an issue for us to evaluate this (either remove the message from all files or add to all of them)?